code/modules/mob/dead/observer/observer_base.dm

```diff
@@ -744,10 +744,15 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 /mob/dead/observer/run_pointed(atom/A as mob|obj|turf in view())
 	if(!..())
 		return FALSE
-	var/follow_link
-	if(invisibility) // Only show the button if the ghost is not visible to the living
-		follow_link = " ([ghost_follow_link(A, src)])"
-	usr.visible_message("<span class='deadsay'><b>[src]</b> points to [A][follow_link].</span>")
+
+	for(var/mob/M in range(7, src))
+		if(M.see_invisible < invisibility)
+			continue //can't view the invisible
+		var/follow_link
+		if(invisibility) // Only show the button if the ghost is not visible to the living
+			follow_link = " ([ghost_follow_link(A, M)])" // Ghost needs to be link clicker, otherwise it breaks
```

Comment:
```suggestion
			follow_link = "([ghost_follow_link(A, M)])" // Ghost needs to be link clicker, otherwise it breaks
```

Reply:
This is actually needed so ` (follow)` shows up optionally after the message. Otherwise it would look like `Baldman points to the table .`

Reply:
Got it, you can resolve the conversation